lib/components/util/dropdown.tsx

```diff
@@ -157,11 +155,12 @@ const Dropdown = ({
   )
 
   return (
-    <DropdownContainer
+    <span
+      className={className}
       id={`${id}-wrapper`}
       onKeyDown={_handleKeyDown}
       ref={containerRef}
-      style={{ float: pullRight ? 'right' : 'left' }}
```

Comment:
I think removing this is causing some issues on mobile

<img width="509" alt="Screenshot 2023-05-19 at 9 45 10 AM" src="https://github.com/opentripplanner/otp-react-redux/assets/86619099/11d7ff69-255b-4b2c-b4c4-d8202f13ddc0">